ocm_python_wrapper/cluster.py

```diff
@@ -377,6 +378,20 @@ def install_addon(
          Returns:
             AddOnInstallation or list: list of stdout responses if rosa is True, else AddOnInstallation
         """
+
+        def _wait_for_rhoam_installation(_command):
+            try:
+                for rosa_sampler in TimeoutSampler(
+                    wait_timeout=TIMEOUT_5MIN,
+                    sleep=SLEEP_1SEC,
+                    func=rosa_cli.execute,
+                    command=_command,
+                ):
+                    return rosa_sampler
```

Comment:
When dubegging rosa sampler is ' ' even when rosa cli resonse stdout is not empty

Reply:
Fixed in [#11](https://github.com/RedHatQE/rosa-python-client/pull/11) to catch WARN in response